cms/djangoapps/contentstore/views/course.py

```diff
@@ -1177,6 +1199,22 @@ def settings_handler(request, course_key_string):
                     elif not entrance_exam_enabled and course_entrance_exam_present:
                         delete_entrance_exam(request, course_key)
 
+                # [COLARAZ_CUSTOM]
+                # Update course links woth secondary organizations.
+                import pdb; pdb.set_trace()
+                new_values = request.json.get('secondary_organizations')
+                if not new_values:
+                    new_values = []
```

Comment:
```suggestion
                new_values = request.json.get('secondary_organizations', [])
```

Reply:
The default value is only useful when there is a missing key. In our case key is there but with value "None" so default value will not work here.

Reply:
nit: In that case, we can do:
`new_values = request.json.get('secondary_organizations') or []`